modules/mobile/spec/request/appointments_vaos_v2_list_request_spec.rb

```diff
@@ -142,6 +159,21 @@
         expect(response.body).to match_json_schema('VAOS_v2_appointments')
         expect(response.parsed_body.dig('data', 0, 'attributes', 'healthcareService')).to be_nil
       end
+
+      it 'attempts to fetch clinic once' do
+        allow_any_instance_of(Mobile::AppointmentsHelper).to receive(:get_clinic).and_return(nil)
```

Comment:
Why are we returning nil? I wonder if this is another area where we'd benefit from `and_call_original`.

Reply:
As per my other comment, I'm not sure why we're returning nil, I was leveraging existing tests as I'm only interested in how many times the function got called. If you'd like this to also re-evaluate if the existing tests make sense though I can do that